### ISSUE TITLE
Include images in toot content for Feed readers

### ISSUE THREAD

**Issue by Krinkle** [mastodon/mastodon#14538]
### Expected behaviour

When opening the Feed for a Mastodon account in a typical feed reader, and viewing the entry for a toot that has one or more images, those images should rendered by the reader as part of the content.

### Actual behaviour

Only the text is shown. Neither the first nor other images are included in the the content or anywhere else in the reader interfaces.

### Steps to reproduce the problem

1. Open <https://polyglot.city/@krinkle.rss> in a common feed reader, such as the NetNewsWire app on iOS or macOS.
2. Observe that in the overview, there is no cover image.
3. Observe that when rendering an entry, there is not a single image.

| Original | Feed reader
|--|--
| <img width="200" alt="Mastodon site" src="https://user-images.githubusercontent.com/156867/86546171-4d52ea80-bf2b-11ea-93e8-74ae841f3dfc.png"> | <img width="200" alt="NetNewsWire" src="https://user-images.githubusercontent.com/156867/86546202-825f3d00-bf2b-11ea-819a-4b5d0393848d.PNG">

### Specifications

* Mastodon v3.2.0.

In searching for related I found, but I'm not sure if these cover this issue (https://github.com/tootsuite/mastodon/issues/7444, https://github.com/tootsuite/mastodon/issues/9286). I also noticed several mentions of an Atom feed, but in the HTML head on profile pages I only found an rss+xml feed and an activity+json feed.

**Comment by Krinkle** [comment]
I'm interested in writing a patch for this if this direction would be accepted as an enhancement.

**Comment by ThibG** [comment]
> I also noticed several mentions of an Atom feed, but in the HTML head on profile pages I only found an rss+xml feed and an activity+json feed.

We used to serve an Atom feed, but it was mainly for OStatus (our former federation protocol) support, not for end-user consumption. When we dropped OStatus support, we dropped atom support as well to reduce code use and avoid having stuff that would be “half-compatible” with OStatus.

Note that RSS feeds already do include attached media in the form of `enclosure` tags. I don't know RSS enough to tell exactly, but https://www.rssboard.org/media-rss#media-content may be more suited than the `enclosure` tag for our purposes.
